apps/saleor-app-checkout/backend/utils.ts

```diff
@@ -69,6 +71,6 @@ export const getSaleorDomain = (): Promise<string> => {
       return reject(`Missing ${envVarsNames.apiUrl} environment variable`);
     }
     const url = new URL(envVars.apiUrl);
-    return resolve(url.hostname);
+    return resolve(url.host);
```

Comment:
Are you sure this is correct?

Reply:
Yes, if we `hostname` this is how `localhost:8000` looks like:
<img width="280" alt="CleanShot 2022-11-09 at 14 12 18@2x" src="https://user-images.githubusercontent.com/4144459/200852649-452edf2f-c021-4ddc-8dc4-768ac7aa4352.png">

We don't have the port that's needed for connection, with `host` we do

Example for domain in cloud:
<img width="321" alt="CleanShot 2022-11-09 at 14 14 27@2x" src="https://user-images.githubusercontent.com/4144459/200853154-e8bb61eb-1768-4ff8-8664-ed19441bc89f.png">